scripts/predict.py

```diff
@@ -10,36 +11,48 @@
 
 @click.command()
 @click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
+              help='Path of json file of the mask')
 @click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
+              help='Path of picture to be predicted')
 @click.option('--model_path', required=True,
-              help='Path where pretrained model')
+              help='Path where pretrained model is located')
 @click.option('--output', default=None,
-              help='Name of output file for predicted output')
-def load_and_predict(mask_path, picture_path, model_path, output):
+              help='Name of file for predicted output')
+@click.option('--img_suffix', default='.png',
+              help='Suffix of predicted image')
+def load_and_predict(mask_path, picture_path, model_path, output, img_suffix):
     for file in (mask_path, picture_path, model_path):
         if not os.path.exists(file):
-            print(file, "Path does not exist")
+            print(file, 'Path does not exist')
             return
+
     if output is None:
-        output = "{}_out.json".format(os.path.splitext(picture_path)[-2])
+        output = '{}_out.json'.format(os.path.splitext(picture_path)[-2])
+    elif not output.endswith('.json'):
+        output = re.sub(img_suffix + '$', '.json', output)
+
+    if not os.path.exists(os.path.dirname(output)):
+        os.makedirs(os.path.dirname(output))
 
-    if os.path.splitext(mask_path)[-1] == ".json":
+    if os.path.splitext(mask_path)[-1] == '.json':
         loader = PKSpaceLoader()
         pictures, _ = loader.load_pic(picture_path, mask_path)
     else:
-        print("Unknown description file")
+        print('Unknown description file')
         return
```

Comment:
@Adman given that we can't assume context from which this script is going to be run I would suggest here printing rather to `stderr` since it is a error and ending with `sys.exit(1)`